lib/generators/acts-as-messageable/migration/migration_generator.rb

```diff
@@ -16,7 +16,9 @@ def self.next_migration_number(dirname)
 
     def create_migration_file
       migration_template 'migration.rb', 'db/migrate/create_messages_table.rb' rescue nil
-      migration_template 'migration_permanent.rb', 'db/migrate/add_recipient_permanent_delete_and_sender_permanent_delete_to_messages.rb' rescue nil
+      migration_template 'migration_permanent.rb',
+        'db/migrate/add_recipient_permanent_delete_and_sender_permanent_delete_to_messages.rb' rescue nil
```

Comment:
Line is too long. [105/80]<br>Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.